index.html

```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
```

Comment:
why we used ???

Reply:
xyz

Reply:
@iam05srk , what's `xyz` ?

Reply:
no that was just random comment while sayyed abdulla ali And mujeeb bhai clearing me the concept of git pull request and its comments and all etc..